core/services/directrequestocr/drlistener.go

```diff
@@ -23,6 +25,56 @@ import (
 var (
 	_ log.Listener   = &DRListener{}
 	_ job.ServiceCtx = &DRListener{}
+
+	promOracleEvents = promauto.NewCounterVec(prometheus.CounterOpts{
+		Name: "functions_oracle_events",
+		Help: "Metric to track oracle events",
+	}, []string{"oracle", "event"})
+
+	promRequestInternalErrors = promauto.NewCounterVec(prometheus.CounterOpts{
+		Name: "functions_request_internal_errors",
```

Comment:
Maybe singular "error" for consistency, since we have success and timeout below, not successes and timeouts.